src/tir/ir/stmt.cc

```diff
@@ -876,17 +876,21 @@ void PrintBlockSignature(const BlockNode* op, ReprPrinter* p) {
   }
 }
 
-void PrintBlockBody(const BlockNode* op, ReprPrinter* p) {
-  // Print init
-  if (op->init.defined()) {
+void PrintInitStmt(const Optional<Stmt>& init, ReprPrinter* p) {
+  if (init.defined()) {
     p->PrintIndent();
     p->stream << "with init() {\n";
     p->indent += 2;
-    p->Print(op->init.value());
+    p->Print(init.value());
```

Comment:
What's it used for?

Reply:
Ahh, it's for printing the an `init` Stmt. To print the `init` Stmt of a SparseBlock, I reused a part of the print function of Block. 

https://github.com/yzh119/tvm/pull/11/files#diff-e2fa840d4ad570e4606dc7654994cc9e4c7fc8e3996a9966879a2b80f7d714f5R879-R896

In ☝️, you can see me introducing a new function `PrintInitStmt(...)` and shortening the original `PrintBlockBody(...)`.

Reply:
okay sounds good